apps/openmw/mwworld/projectilemanager.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <string>
 
+#include <osg/observer_ptr>
```

Comment:
What version of GCC or Clang are you using?

I'm able to use both without a problem with Debian userland.